src/components/Bale/BaleList.js

```diff
@@ -82,15 +59,25 @@ class BaleList extends Component {
 }
 
 BaleList.propTypes = {
+  fetchData: PropTypes.func.isRequired,
+  token: PropTypes.string,
+  bales: PropTypes.object,
   openEditBaleModal: PropTypes.func.isRequired,
 };
 
-BaleList.defaultProps = {};
+BaleList.defaultProps = {
+  token: false,
+  bales: [],
+};
 
-const mapStateToProps = () => ({});
+const mapStateToProps = state => ({
+  token: state.login.token,
+  bales: state.bales.bales,
```

Comment:
You made the selector function but you're not using it. Either remove the selector, or change this line. I'd recommend using it for code consistency